lib/magma/commands.rb

```diff
@@ -2,6 +2,42 @@
 require 'logger'
 
 class Magma
+  class LoadProject < Etna::Command
+    def execute(file_name)
+      file = File.open(file_name)
+      file_data = JSON.parse(JSON.parse(file.read))
+
+      db = Magma.instance.db
+
+      models = file_data["models"]
+
+      models.keys.each do |model|
+        model_name = models[model]["template"]["name"]
+        attributes = models[model]["template"]["attributes"]
+
+        attributes.keys.each do |att|
+          attribute = attributes[att]
+
+          db[:attributes].insert(
+            project_name: 'i dont know where to find the project name',
+            model_name: model_name,
+            attribute_name: attribute["name"],
+            type: attribute["type"] || "project",
+            display_name: attribute["display_name"],
+            # attribute["shown"] is the opposite of hidden?
+            hidden: attribute["shown"],
+            validation: attribute["validation_object"],
+            restricted: attribute["restricted"],
+            read_only: attribute["read_only"],
+            link_model_name: attribute["link_model_name"]
+            # where does attribute_class go?
```

Comment:
`attribute_class` is only used by Timur. I think it's mostly used to decide how different attributes should be rendered. It doesn't need to be inserted into the `attributes` table.

Reply:
We'll deprecate it shortly anyway, since the `attribute_type` (which should be in here) should replace it.